### ISSUE TITLE
Todo: First draft of roadmap

### ISSUE THREAD

**Issue by Protohedgehog** [open-science-academy/Orga#16]
See roadmap.md for details

**Comment by wolass** [comment]
Thoughts for the roadmap:
SA should be seen as a teaching organisation with expertise on open and reproducible science. Our project nr 1 would be the teaching and certification of scientists using the mooc.'

2nd project for the FSA as I see it is the evaluation of scientists and science using an alternative measure. And here we may argue that it is going to be another impact factor. But we need to account for project 3. 

Which is the nano-publication platform. So that our measure will be compatible with the current publishing standards and also with the new platform. It's going to be challenging but it might be a combination that is unique and complete. Do you also see another use case for the FSA?

**Comment by Protohedgehog** [comment]
Agreed about teaching. In a P2P learning framework. I am not sure about the status of the MOOC, but given there has not been any development in around 3 months now, and the platform is still dead (Eliademy), I am not sure how much we can depend on that. We can certainly re-use all existing materials.

Thinking aloud on that. There are about 2.8/10 modules completed at the present. The latter .8 on Open Access, which I can complete independently, as I was the main contributor to that module anyway. Each module can be eg transformed into an e-book too. Or the whole thing in the end. I think the OS MOOC did start that, but status is the same as the rest of the project.

Evaluation. I like it, because it gears with the practical and political and community dimensions of the FSA. We will be going up against/with the OS Monitor though too. It needs substantial programming knowledge, I think. 

There is a massive market potential at the moment for 'open metrics', which is still in its infancy and expected to grow (see eg INORMS and the OSM). There is a demand for metrics service providers, as most institutes do a shitty job of recording eg their OA publications. You can provide an economic reasoning for this too; for example, for every publication, some institutions will receive some money from a governmental agency or ministry. For every OA publication, the institution will receive the money multiplied by a factor. It mandates institutes to become economically interested in keeping efficient OA records, as one example. Can be done at the author-level too, and leverage things like Unpaywall/ImpactStory. Launch it as a non-profit project under the FSA. Imagine institutions proudly showing their OA scores and competing for funding based on a measurable openness of their research. So, let them be gamed.
